hpcc-aks/main.tf

```diff
@@ -1,10 +1,14 @@
+variable "subscription_id" {
+  default = "example"

```

Comment:
thought... 
Might be better to not set any default. The reason for that is 
 1. With default, when the repo is cloned and run "terraform plan", it would throw an error saying there is no such subscription "example". 
2. W/o default, it would ask for the variable value before it even runs.

Reply:
Actually, I took that back (above comment). Disregard it. I just noticed the whole thing is supposed to be an example.